webapp/src/ts/modals/edit-user/update-password.component.html

```diff
@@ -0,0 +1,41 @@
+<mm-modal
+  status="status"
```

Comment:
Change to `[status]="status"`, as we need one-way binding.

Reply:
Fixed.